.cz-config.js

```diff
@@ -25,5 +25,6 @@ module.exports = {
 		{ value: "cli", name: "cli:		Core CLI things" },
 		{ value: "misc", name: "misc:		Other formats like tweaks and such" },
 		{ value: "chore", name: "chore:	Updating docs, linting, etc" },
+		{ value: "fix", name: "fix: it fixes a small bug" }
```

Comment:
I'd change the description to be more general like the others

Reply:
Done!